src/containers/cooperation-details/cooperation-activities/CooperationActivities.tsx

```diff
@@ -114,7 +100,11 @@ const CooperationActivities: FC<CooperationActivitiesProps> = ({
         >
           {t('common.cancel')}
         </AppButton>
-        <AppButton size={SizeEnum.ExtraLarge} type={ButtonTypeEnum.Submit}>
+        <AppButton
+          onClick={() => void updateCooperationSection()}
```

Comment:
Do we need a void here ?

Reply:
Unfortunately yes, cause AppButton awaits for void, while updateCooperationSection is a promise